docs/src/overview/get-support.md

```diff
@@ -0,0 +1,40 @@
+---
+title: " Get support"
+weight: 4
+toc: false
+description: |
+  Find out how to get help If you're experiencing an issue with Platform.sh.
+---
+
+{{< description >}}
+
+
+## Create a support ticket
+
+1. [Open the Management Console](https://console.platform.sh/)
+2. Select **User menu** (your name) > **Support**.
+3. Click **+ New ticket**.
+4. Fill in the ticket fields and click **Submit**.
+
+In the future, you can [access the ticket list](https://console.platform.sh/-/users/~/tickets) or [open a ticket directly](https://console.platform.sh/-/users/~/tickets/open) using the shortcuts.
+
+Once you submit a ticket, you can see it on the list alongside tickets created by users in your organization over time.
+Note that once you submit the ticket you won't be able to modify or delete the submission.
+If you have any additional information, you can select the submitted ticket and write a message.
+
+
+
+## Slack
+
+If you want to chat with engineers and other customers you can join the [public Slack channel](https://chat.platform.sh/).
+
+## Community
+
+In the [Platform.sh Community Site](https://community.platform.sh/) you can find plenty of how-to guides that contains suggestions on how to make the most out of Platform.
+
+
+## Contact Sales
+
+If you have questions with regards pricing or if you need help figuring out if Platform.sh is the right solution for your team, get in touch with [Sales](https://platform.sh/contact/).
```

Comment:
```suggestion
If you have questions about pricing or need help figuring out if Platform.sh is the right solution for your team, get in touch with [Sales](https://platform.sh/contact/).
```

It could be "questions with regards to" or "questions regarding", but they're both too formal (especially the first) and not as easily understood.